hosting_docs/source/changelog/0058-RabbitMQ_upgrade_to_3.10.7_version.md

```diff
@@ -0,0 +1,25 @@
+<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
+<!--See https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md for instructions-->
+# 58. RabbitMQ upgrade to 3.10.7 version
+
+**Date:** 2022-10-30
+
+**Optional per env:** _only required on some environments_
+
+
+## CommCare Version Dependency
+This change is not known to be dependent on any particular version of CommCare.
+
+
+## Change Context
+This change upgrade RabbitMQ 3.10.7 version. 
+The current installed RabbitMQ version 3.8.5 is already End of Life. 
+So, we strongly recommend applying this change. 
+
+## Details
+As part of our ongoing effort to keep CommCare HQ up to date with the latest tools and 
+libraries we have updated RabbitMQ to 3.10.7 version. 
+
+## Steps to update
+1. For users running RabbitMQ on Linux machines:
+   instructions: [RabbitMQ_upgrade](https://commcare-cloud.readthedocs.io/en/latest/services/rabbitmq/upgrade.html?highlight=rabbitmq)
```

Comment:
(Fine to leave this as for now, but for future reference) It is better to refer to these using internal references/links without the hostname, so that if the host changes in future we don't need to update these.